docs/buildah-config.md

```diff
@@ -180,7 +180,7 @@ If names are used, the container should include entries for those names in its
 
 **--volume** *volume*
 
-Add a location in the directory tree which should be marked as a *volume* in any images which will be built using the specified container. Can be used multiple times.
+Add a location in the directory tree which should be marked as a *volume* in any images which will be built using the specified container. Can be used multiple times. If *volume* has trailing `-`, and does not exist on disk, then the volume is removed from the config.
```

Comment:
Ugh, hope I didn't cause heartburn with a bad comment.  Can you s/has trailing/has a trailing/ please?

Reply:
Done, I should be more careful when reading